libqtile/confreader.py

```diff
@@ -58,6 +58,7 @@ class Config:
     dgroups_app_rules: list[Rule]
     follow_mouse_focus: bool | Literal["click_or_drag_only"]
     focus_on_window_activation: Literal["focus", "smart", "urgent", "never"] | FunctionType
+    focus_previous_on_window_remove: bool
```

Comment:
This will need to be added to the default config. You should also update the page in the docs that talks about the config variables and add a description of this one.